app/src/org/commcare/dalvik/activities/DispatchActivity.java

```diff
@@ -0,0 +1,311 @@
+package org.commcare.dalvik.activities;
```

Comment:
Does this have an associated layout/view? It will be shown as the parent activity if, say, the corruption dialog is popped up, so it seems like it should have some sort of associated background view (Like Aliza's layout that is shown when commcare is switching between seated apps.